java/xgboost4j/src/main/resources/lib/README.md

```diff
@@ -0,0 +1 @@
+please put native library in this package.
```

Comment:
can we auto generate the lib to this path? or let native lib search for possible path? see https://github.com/dmlc/xgboost/blob/master/wrapper/xgboost.py#L58